guide/process/steps/task.md

```diff
@@ -52,44 +53,67 @@ By default, the task's inputs are the previous step's outputs. Can be customized
 
 ## Input
 
-Inputs can be any type of data:
+Inputs can be any of the following type:
 - `string`
 - `bool`
 - `number`
 - `array`
 - `object`
 - `reference`
 
-### Composable type (`array` or `object`)
+### Constant
```

Comment:
Looks like `Composable` are not constants in that case. If you see how to improve that quickly that would be great otherwise it's fine.

Reply:
I moved it at last paragraph and add an example with reference.